i6_models/parts/frontend/__init__.py

```diff
@@ -0,0 +1 @@
+from .vgg_act import *
```

Comment:
Is this a good idea? I think I would rather not have this.
```suggestion
```

Reply:
Why is this not good?

Reply:
Why would you want that? There could be other frontends. Do you really want to put all possible frontends into the `i6_models.parts.frontend` namespace?

For the same reason, we don't have `i6_models.assemblies.ConformerEncoderV1`, you need to import it from `i6_models.assemblies.conformer.ConformerEncoderV1`.

Reply:
Flexibility for the user.

Reply:
But I have no strong feelings on this